packages/core/admin/admin/src/utils/unique-admin-hash.js

```diff
@@ -0,0 +1,7 @@
+const crypto = require('crypto');
```

Comment:
crypto is a node package you don't want to use this in the frontend. Look into an equivalent with he browser APIs here